docs/api-usage/prices.mdx

```diff
@@ -3,6 +3,7 @@ title: Prices
 ---
 
 Saleor API has four different types that represent prices. The most basic type consists of a currency code and a numeric amount. The most complex one can define a range of prices, including net, gross, and tax amounts.
+The numeric amount precision is determined by the precision of the price currency.
```

Comment:
I would suggest adding an example:

 The numeric amount precision is determined by the precision of the price currency. For example, USD has a precision value of 2, such as $1.00